srcpkgs/binutils/template

```diff
@@ -18,6 +18,7 @@ else
 	unset XBPS_CHECK_PKGS
 	XBPS_TRIPLET=x86_64-unknown-linux-gnu
 fi
+hostmakedepends+=" texinfo"
```

Comment:
This can't be added here because `texinfo` is not a bootstrap package.
Instead it should be set in the `if [ "$CHROOT_READY" ];` block above.